<launch>

    <node name="Mavros_data_sim" pkg="rtk_data_trans" type="Mavros_data_sim" output="screen">
        <remap from="/mavros/global_position/global" to="sim/mavros/global_position/global"/>
        <remap from="/mavros/global_position/rel_alt" to="sim/mavros/global_position/rel_alt"/>
        <remap from="/mavros/gpsstatus/gpsl/raw" to="sim/mavros/gpsstatus/gpsl/raw"/>
    </node>

    <node name="RTK_data_sender" pkg="rtk_data_trans" type="RTK_data_sender" output="screen">
        <!-- <param name="endpoints" value="192.168.3.217, 14500; 127.0.0.1, 14552; 192.168.3.188, 14550; 192.168.3.129, 14552"/> -->
        <param name="endpoints" value="127.0.0.1, 14560;"/>
        <remap from="/mavros/global_position/global" to="sim/mavros/global_position/global"/>
        <remap from="/mavros/global_position/rel_alt" to="sim/mavros/global_position/rel_alt"/>
        <remap from="/mavros/gpsstatus/gpsl/raw" to="sim/mavros/gpsstatus/gpsl/raw"/>
    </node>

    <node name="RTK_data_receiver" pkg="rtk_data_trans" type="RTK_data_receiver" output="screen">
        <param name="remote_port" value="14560"/>
        <remap from="/new/gpsfix" to="/rtk_land_pos/global_position/global"/>
    </node>

</launch>
